kstars/fitsviewer/fitsview.cpp

```diff
@@ -782,12 +782,46 @@ void FITSView::drawStarCentroid(QPainter * painter)
 
     foreach (auto const &starCenter, imageData->getStarCenters())
     {
```

Comment:
I see this block will conflict with https://phabricator.kde.org/D28767. We need to come up with a better solution, perhaps moving the drawing code to the detector too. OK for now.

Reply:
I saw that there were some merge conflicts in this method. I will resolve them too.